shared/lib/TemplatesDeployer.js

```diff
@@ -0,0 +1,204 @@
+const { APPS } = require('../helpers/apps')
+const { hash: namehash } = require('eth-ens-namehash')
+
+const logDeploy = require('@aragon/os/scripts/helpers/deploy-logger')
+const deployAPM = require('@aragon/os/scripts/deploy-apm')
+const deployENS = require('@aragon/os/scripts/deploy-test-ens')
+const deployAragonID = require('@aragon/id/scripts/deploy-beta-aragonid')
+const deployDAOFactory = require('@aragon/os/scripts/deploy-daofactory')
```

Comment:
We've been shifting away from bundling these deployment scripts as part of individual repos, but I'm interested in what you think about our current strategy:

- Remove deployment scripts from most Aragon-related repos, unless they're optional dependencies in a general Aragon environment (e.g. the test tokens)
- Place the deployment scripts in [aragen](https://github.com/aragon/aragen), which should be able to spin up either a new chain, or deploy a base Aragon environment to an existing chain

We may want to start changing the script dependencies to be those in aragen, or we can wait until we finalize the migration (longer term).